mobile/src/reducers/calendar/calendar-events.reducer.ts

```diff
@@ -84,11 +89,13 @@ export const calendarEventsReducer: Reducer<CalendarEventsState> = (state = init
                 disableCalendarActionsButtonGroup: false
             };
         case 'SELECT-CALENDAR-DAY':
-            const newSickLeaveEventState = selectSickLeaveEndDateReducer(state, action);
+            const startSickLeaveDialog = selectStartDateSickLeaveReducer(state, action);
```

Comment:
are we going to have to change this method every time we add another 'select something' dialog?

Reply:
yes

Reply:
this does look weird... Ideally, these dialogs should not intersect and calendar should not really know about sick leave events. It should be done somewhere else, and some epic should translate events like CONFIRM_START-DATE-SICK-LEAVE to something like 'NEW-INTERVAL-ADDED'. And it's probably should not happen immediately, I'd wait for HTTP request to finish and THEN send this message